### DISCUSSION TITLE
Feature for NOT Showing NSFW pics

### DISCUSSION THREAD

**Discussion by ash-bluepollution95** [BlafKing/sd-civitai-browser-plus#262] [Ideas]
Consider this:
3:30pm weekday.
You're babysitting your honorary niece and nephew and they come ROLLING IN and seeing things they're not meant to (THANKFULLY THIS DOESNT HAPPEN XD - it's a bad example i know lmao) 

Basically saying:
I love the gallery aspect, but i'd love the option to NOT see the NSFW pics up front.
Considering i've been working a lot with PonyXL lately my brain has seen enough XXX for one day :P

**Comment by BlafKing** [comment]
There's a toggle in the filter box to disable NSFW models, or did you mean disabling NSFW images somewhere else?

**Comment by ash-bluepollution95** [comment]
Well the hard call is most PDXL models are NSFW in nature, but have a PG or
PG 13 rated image somewhere in the model.
More so i was just curious if it could do that?
If NOT, then look i'll just live with it :P
Your extension is amazing, and considering that Link went desktop and i'm
largely server based -- this is just a A+ toy
Kieran Somerville aka
"The Duskfall Portal Crew"
Artist / Indie Music / Designer / Ai Artist
+64 22 678 8131
***@***.***
https://www.end-media.org
"Do not print this email, a tree will eat you for breakfast."

**Comment by BlafKing** [comment]
Thanks for clarifying!
Before it used to blur images that were NSFW when the NSFW filter was turned off in the filter box, I see now that this behavior no longer works 😅 I'll make sure to include a fox for this in the next update!

The behavior *should* be that when the NSFW Toggle is turned off, then it should not show any models that itself are tagged NSFW, and if a "non" NSFW model has NSFW images as preview images, then it should automatically blur those images so that they are not visible.
(and the blurring is currently broken like I said)

Thanks for bringing it to my attention, it's added to the to-do list! :)

**Comment by ash-bluepollution95** [comment]
Amazing!
Thank you so much, also the issue i ran into that I closed is like a
server-gpu only issue and I figured out that if i install "pip fake-user
agent" Or something, it's like isolated to the way I use vast -- because I
have to pre-install many things that somehow get missed (I can't gen
locally sadly XD)
Kieran Somerville aka
"The Duskfall Portal Crew"
Artist / Indie Music / Designer / Ai Artist
+64 22 678 8131
***@***.***
https://www.end-media.org
"Do not print this email, a tree will eat you for breakfast."
